testsuite/MDAnalysisTests/formats/test_libmdaxdr.py

```diff
@@ -129,78 +129,86 @@ class Test_XTCFile(XDRFormatBaseTest):
 
 def test_xtc():
     f = XTCFile(XTC_single_frame)
-    assert f.n_atoms == 10
+    assert_equal(f.n_atoms, 10)
     xyz, box, step, time, prec = f.read()
     # xtc only saves with 3 decimal places precision
     assert_array_almost_equal(xyz.flat, np.arange(30), decimal=3)
     assert_array_almost_equal(box, np.eye(3) * 20, decimal=3)
-    assert step == 0
-    assert time == 10.0
-    assert prec == 1000.0
+    assert_equal(step, 0)
+    assert_equal(time, 10.0)
+    assert_equal(prec, 1000.0)
 
 
 def test_read_multi_frame_xtc():
     with XTCFile(XTC_multi_frame) as f:
         f.seek(5)
-        assert f.tell() == 5
-        assert len(f) == 10
-        assert f.tell() == 5
+        assert_equal(f.tell(), 5)
+        assert_equal(len(f), 10)
+        assert_equal(f.tell(), 5)
         f.seek(0)
-        assert f.tell() == 0
+        assert_equal(f.tell(), 0)
         ones = np.ones(30).reshape(10, 3)
         box_compare = np.eye(3) * 20
         for i, frame in enumerate(f):
-            print(i)
             xyz, box, step, time, prec = frame
             assert_array_almost_equal(xyz, ones * i, decimal=3)
             assert_array_almost_equal(box, box_compare, decimal=3)
-            assert step == i
-            assert time == i * .5
-            assert prec == 1000.0
+            assert_equal(step, i)
+            assert_equal(time, i * 0.5)
+            assert_equal(prec, 1000.0)
 
 
 @run_in_tempdir()
 def test_write_xtc():
     with XTCFile(XTC_multi_frame) as f_in:
         with XTCFile('foo.xtc', 'w') as f_out:
-            assert 0 == f_out.n_atoms
+            assert_equal(f_out.n_atoms, 0)
             for frame in f_in:
                 f_out.write(*frame)
 
     with XTCFile('foo.xtc') as f:
-        assert len(f) == 10
+        assert_equal(len(f), 10)
         ones = np.ones(30).reshape(10, 3)
         box_compare = np.eye(3) * 20
         for i, frame in enumerate(f):
-            print(i)
             xyz, box, step, time, prec = frame
             assert_array_almost_equal(xyz, ones * i, decimal=3)
             assert_array_almost_equal(box, box_compare, decimal=3)
-            assert step == i
-            assert time == i * .5
-            assert prec == 1000.0
+            assert_equal(step, i)
+            assert_equal(time, i * 0.5)
+            assert_equal(prec, 1000.0)
 
 
-@raises(ValueError)
 @run_in_tempdir()
-def test_write_different_box_xtc():
+def test_different_box_xtc():
```

Comment:
Out of the context of the pull request, it is not completely obvious what the test is about from its name. A docstring could help. Note that only the first line is displayed in the verbose mode of nose.